packages/labs/virtualizer/README.md

```diff
@@ -431,6 +431,13 @@ Type: `(item: T, index?: number) => TemplateResult`
 
 A function that returns a Lit `TemplateResult`. It will be used to generate a child element for each item in the `items` array.
 
+### `keyFunction` property
+
+Type: `(item: T, index: number) => unknown`
+Generic type name: `KeyFn` from `lit/directives/repeat.js`
+
+A function given to the `repeat` directive. This is given the item value and it should return a guaranteed unique key. The unique key helps the repeat directive optimize rendering of large lists.
```

Comment:
Also, I described the input and return value purely since some users of the virtualizer may not have used (or remember) the `repeat` directive usage. Maybe this could actually just link out to those docs. That way there is no duplication of the source content.

Reply:
My point about mentioning `repeat` is that this is a somewhat out of the blue reference to it. Nowhere else do we say that virtualizer uses repeat, and it's entirely an implementation detail. virtualizer could in the future use it's own DOM movement utility.

So I'd rather not mention repeat at all and just document what the key function does from the perspective of virtualizer's behavior.

Reply:
Ah ok. I get what you mean now. I'll work on updating the text in a little bit to do that. I think I was trying to not "re-invent the wheel". But yea, it isn't necessarily a detail that consumers of the virtualizer need to know about.